packages/local_auth/local_auth_darwin/lib/types/auth_messages_darwin.dart

```diff
@@ -6,12 +6,12 @@ import 'package:flutter/foundation.dart';
 import 'package:intl/intl.dart';
 import 'package:local_auth_platform_interface/types/auth_messages.dart';
 
-/// Class wrapping all authentication messages needed on iOS.
+/// Class wrapping all authentication messages needed on iOS & MacOS.
 /// Provides default values for all messages.
 @immutable
-class IOSAuthMessages extends AuthMessages {
+class DarwinAuthMessages extends AuthMessages {
```

Comment:
See https://github.com/flutter/packages/pull/5766#issuecomment-1878641342 and https://github.com/flutter/packages/pull/5766#discussion_r1486398912

Reply:
(Also, renaming public classes is a breaking change, which this doesn't otherwise need to be.)

Reply:
I put back iOSAuthMessages and created MacOSAuthMessages